usi/src/main/proto/usi.proto

```diff
@@ -65,15 +74,12 @@ message SwitchInfo {
   // Telnet Port
   int32 telnet_port = 2;
 
-  // Device Port
-  int32 device_port = 3;
-
   // Switch model
-  SwitchModel model = 4;
+  SwitchModel model = 3;
 
   // Switch connect username
-  string username = 5;
+  string username = 4;
```

Comment:
NB: You don't need to renumber the entries if you don't want to -- it doesn't hurt anything, but not necessary. I think Yufeng said that the only impact it has is on the ordering of the serialization in the API view.  In some circumstances (not ours), it's actually explicitly intended that the numbers *don't* change!